apps/condo/domains/news/schema/GetNewsItemsRecipientsCountersService.js

```diff
@@ -61,7 +61,7 @@ const GetNewsItemsRecipientsCountersService = new GQLCustomSchema('GetNewsItemsR
                                 orConditions.push(...queryConditionsByUnits(property))
                             }
 
-                            return []
+                            return chunk
```

Comment:
The result of `loadListByChunks` is not using. Chunk processor is used here to reduce memory consumption.
Looks like a bug within of `loadListBuChunks`. We need to increase the `skip` variable before start chunkProcessor.

Reply:
Refactored